posthog/batch_exports/http.py

```diff
@@ -354,6 +354,8 @@ class BatchExportViewSet(TeamAndOrgViewSetMixin, viewsets.ModelViewSet):
     scope_object = "batch_export"
     queryset = BatchExport.objects.exclude(deleted=True).order_by("-created_at").prefetch_related("destination").all()
     serializer_class = BatchExportSerializer
+    scope_object_read_actions: list[str] = ["retrieve", "list"]
+    scope_object_write_actions: list[str] = ["destroy", "create", "backfill", "pause", "unpause"]
```

Comment:
A nicer way of doing this tbh is with the `@action` decorator adding `required_scopes=["batch_export:read"]` as that way its inline with the action

Reply:
Although this also works so will approve